package.json

```diff
@@ -17,7 +17,7 @@
     "test:single": "mocha --require withDom.js --compilers js:babel-core/register --watch --reporter dot",
     "test:watch": "mocha --require withDom.js --compilers js:babel-core/register --recursive test/*.js --watch --reporter dot",
     "test:env": "sh ./example-test.sh",
-    "test:all": "npm run react:13 && npm test && npm run react:14 && npm test && npm run react:15 && npm test",
+    "test:all": "npm run lint && npm run react:13 && npm run test:only && npm run react:14 && npm run test:only && npm run react:15 && npm run test:only",
```

Comment:
for a nicer hook, let's run `npm run lint` in "pretest" instead of `test`, and then here, we can `npm run pretest` everywhere we currently run `npm run lint` (that's not "test")?